src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.ts

```diff
@@ -342,7 +342,7 @@ export class TutorExerciseDashboardComponent implements OnInit {
                 route = `/modeling-exercise/${this.exercise.id}/submissions/${submission}/assessment`;
                 break;
             case ExerciseType.FILE_UPLOAD:
-                route = `/file-upload-exercise/${this.exercise.id}/submissions/${submission}/assessment`;
+                route = `/file-upload-exercise/${this.exercise.id}/submission/${submission}/assessment`;
```

Comment:
Wasn't `submissions` correct here?

Reply:
For client routes we use singular, not plural. This will be also changed for modeling and text exercises later

Reply:
Ah, you are right!